src/components/PeopleManagement/data/hooks/useLearnerCreditPlans.ts

```diff
@@ -0,0 +1,42 @@
+import { useQuery } from '@tanstack/react-query';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+
+import EnterpriseAccessApiService from '../../../../data/services/EnterpriseAccessApiService';
+import { peopleManagementQueryKeys } from '../../constants';
+
+interface LearnerCreditPlan {
```

Comment:
Would recommend exporting this and just importing in LearnerAccess.tsx (if we don't put in a general-purpose spot like `types.d.ts`) 
```suggestion
export interface LearnerCreditPlan {
```